packages/gatsby-transformer-csv/src/gatsby-node.js

```diff
@@ -49,7 +52,9 @@ async function onCreateNode(
   function transformObject(obj, i) {
     const csvNode = {
       ...obj,
-      id: obj.id ? obj.id : createNodeId(`${node.id} [${i}] >>> CSV`),
+      id: obj.id
+        ? obj.id
```

Comment:
This can be `??` as well (if you're going to change this line anyways :) )

Reply:
I'm cool with that. Consistency is nice.